Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rust:bullseye as build
```

Comment:
We already have a similar Dockerfile in `dockerfiles/`

Reply:
we should follow the same pattern as other repos and. that is to keep the `Dockerfile` at the root of the project.

Reply:
I like the idea of keeping the project root nice an clean, we will also probably end up with multiple dockerfiles (debian, alpine, etc). 
The `dockerfiles` directory convention is also adopted in our other repos (https://github.com/hirosystems/clarinet, https://github.com/hirosystems/chainhook).